Sources/Contentful/Client.swift

```diff
@@ -473,7 +464,7 @@ open class Client {
         } catch let error {
             let sdkError = SDKError.unparseableJSON(data: data, errorMessage: "\(error)")
             ContentfulLogger.log(.error, message: sdkError.message)
-            completion(.failure(sdkError))
+            return .failure(sdkError)
```

Comment:
Another example of where function execution would continue after calling `completion`, resulting in `completion` being called twice.